Gordon360/Services/ProfileService.cs

```diff
@@ -347,6 +348,27 @@ public async Task<FacultyStaffProfileViewModel> UpdateOfficeLocationAsync(string
             return profile;
         }
 
+        /// <summary>
+        /// office hours setting
+        /// </summary>
+        /// <param name="username"> The username for the user whose office hours is to be updated </param>
+        /// <param name="newHours">The new hours to update the user's office hours to</param>
+        /// <returns>updated fac/staff profile if found</returns>
+        public async Task<FacultyStaffProfileViewModel> UpdateOfficeHoursAsync(string username, string newHours)
+        {
+            var profile = GetFacultyStaffProfileByUsername(username);
+            if (profile == null)
+            {
+                throw new ResourceNotFoundException() { ExceptionMessage = "The account was not found" };
+            }
+            var acccount = _webSQLContext.accounts.Where(a => a.AD_Username == username).FirstOrDefault();
```

Comment:
You can simplify this with the filtering overload of `FirstOrDefault`:
```suggestion
            var acccount = _webSQLContext.accounts.FirstOrDefault(a => a.AD_Username == username);
```